test/e2e/instrumentation/logging/utils/wait.go

```diff
@@ -150,33 +150,34 @@ func getFullIngestionPred(podsMap map[string]FiniteLoggingPod) NumberedIngestion
 	return func(name string, occ map[int]bool) (bool, error) {
 		p := podsMap[name]
 		ok := len(occ) == p.ExpectedLineCount()
-		if !ok {
-			framework.Logf("Pod %s is still missing %d lines", name, p.ExpectedLineCount()-len(occ))
-		}
 		return ok, nil
 	}
 }
 
 func getFullIngestionTimeout(podsMap map[string]FiniteLoggingPod, slack float64) NumberedTimeoutFun {
 	return func(names []string, occs map[string]map[int]bool) error {
 		totalGot, totalWant := 0, 0
-		podsWithLosses := []string{}
+		lossMsgs := []string{}
 		for _, name := range names {
 			got := len(occs[name])
 			want := podsMap[name].ExpectedLineCount()
 			if got != want {
-				podsWithLosses = append(podsWithLosses, name)
+				lossMsg := fmt.Sprintf("%s: %d lines", name, want-got)
+				lossMsgs = append(lossMsgs, lossMsg)
 			}
 			totalGot += got
 			totalWant += want
 		}
-		if len(podsWithLosses) > 0 {
-			framework.Logf("Still missing logs from: %s", strings.Join(podsWithLosses, ", "))
+		if len(lossMsgs) > 0 {
+			framework.Logf("Still missing logs from:\n%s", strings.Join(lossMsgs, "\n"))
 		}
 		lostFrac := 1 - float64(totalGot)/float64(totalWant)
```

Comment:
Theoretically -- yes, it's possible to write such code, in practice -- no, it doesn't make sense